### PR TITLE
Add support to ingressClassName

### PR DESCRIPTION
**Pull request by caiohasouza**
Fix https://github.com/datalust/helm.datalust.co/issues/24

### CODE REVIEW

File: charts/seq/templates/ingress.yaml [hunk 1]
```diff
@@ -27,6 +27,9 @@ spec:
       secretName: {{ .secretName }}
 {{- end }}
 {{- end }}
+  {{- if .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
```
Comment (KodrAus) [inline charts/seq/templates/ingress.yaml:31]: Since this lives under the `ingress` object what do you think about just calling this `className`?

**Review by KodrAus — COMMENTED** [review]
Thanks for working on this @caiohasouza! I just had one minor naming comment

File: charts/seq/templates/ingress.yaml (hunk 1 above)
Comment (caiohasouza) [reply charts/seq/templates/ingress.yaml:31]: Hi @KodrAus,

I normally use the same Kubernetes Object name on Helm variables to be clear and easy to understand, this practice is adopted on many public helm charts, like Grafana, Jenkins, etc. But you do you think that className is better, Ok, i change. just confirm please.

Regards

Comment (KodrAus) [reply charts/seq/templates/ingress.yaml:31]: I don't think that's a practice we've really followed so far so I think we should go with `className` for now. At some point in the future we can do a complete pass over the API we've evolved here and try make it consistent with patterns used by popular charts.

Comment (caiohasouza) [reply charts/seq/templates/ingress.yaml]: Ok, no problems, change done!

File: charts/seq/values.yaml [hunk 2]
```diff
@@ -70,6 +70,7 @@ service:
   type: ClusterIP
 
 ingress:
+  #className: nginx
```
Comment (KodrAus) [inline charts/seq/values.yaml:73]:
```suggestion
  # className: nginx
```

Comment (caiohasouza) [reply charts/seq/values.yaml]: Done!

**Comment by KodrAus** [comment]
Thanks @caiohasouza! We do need to try follow established patterns better here, if you've got any other charts in mind besides the few you've mentioned that you think set good precedents we should look to I'd be keen to check them out!

**Review by KodrAus — APPROVED** [review]

**Comment by caiohasouza** [comment]
Perfect, thank you @KodrAus

**Comment by caiohasouza** [comment]
@KodrAus , 

This feature is not released yet on 2021.4.7192 helm chart version, right? I'm trying to use but the changes do not take effetc. If yes it's possible generate a new helm chart version to include this feature?

Regards

**Pull request merged**